mtp_api/apps/security/tests/test_views.py

```diff
@@ -1076,18 +1078,63 @@ def test_check_filtering_by_credit_resolution(self):
             Check.objects.filter(status=CHECK_STATUS.PENDING).count() - 1,
         )
 
+    def test_check_filtering_by_actioned_by(self):
+        """
+        Test that the list endpoint only returns the checks with an actioned_by id.
+        """
+        filters = {
+            'actioned_by': True,
+        }
+
+        auth = self.get_http_authorization_for_user(self._get_authorised_user())
+        response = self.client.get(
+            reverse('security-check-list'),
+            filters,
+            format='json',
+            HTTP_AUTHORIZATION=auth,
+        )
+
+        self.assertEqual(response.status_code, http_status.HTTP_200_OK)
+
+        response_data = response.json()
+
+        self.assertEqual(
+            response_data['count'],
+            Check.objects.filter(actioned_by__isnull=False).count(),
+        )
+
 
 class GetCheckTestCase(BaseCheckTestCase):
     """
     Tests related to getting a single security check.
     """
 
-    def test_get(self):
+    def test_get_for_a_non_actioned_check(self):
         """
         Test that the get object endpoint returns check details.
         """
         check = Check.objects.first()
```

Comment:
how do you know this is "non actioned"?

Reply:
actually, what's the difference between `test_get_for_a_non_actioned_check` and `test_get_for_an_actioned_check`?

Reply:
test_get_for_a_non_actioned_check = actioned_by is null
test_get_for_an_actioned_check = actioned_by is not null

Reply:
I have changed this line to `check = Check.objects.filter(actioned_by__isnull=True).first()`

Reply:
but this endpoint doesn't do any filtering

Reply:
This now makes it more explicit on what the test means imo

Reply:
no harm i guess, but i can't see what difference in logic you're testing

Reply:
Oh I see what you're saying 🥇

Reply:
My misunderstanding